tests/test_seed.py

```diff
@@ -0,0 +1,66 @@
+from opensoundscape import seed
+import numpy as np
+import torch
+import random
+from opensoundscape.ml import cnn, cnn_architectures
+
+import pytest
+
+pytestmark = pytest.mark.parametrize("input", [1, 11, 13, 42, 59, 666, 1234])
```

Comment:
what does this do?

Reply:
This is a global version of the @pytest.mark.parametrize decorator to use the list as inputs for all functions in the script. See https://docs.pytest.org/en/7.3.x/how-to/parametrize.html for more explanation.